go.mod

```diff
@@ -158,3 +159,7 @@ require (
 
 // replaced due to https://github.com/satori/go.uuid/issues/73
 replace github.com/satori/go.uuid => github.com/gofrs/uuid v3.2.0+incompatible
+
+replace github.com/golang/protobuf => github.com/golang/protobuf v1.3.3
+
+replace google.golang.org/genproto => google.golang.org/genproto v0.0.0-20200218151345-dad8c97a84f5
```

Comment:
Is this really required?

Reply:
This is required, unfortunately, due to incompatibilities between different versions of the `protobuf` and `grpc` packages. I realize Telegraf currently uses `1.3.5` and not `1.3.3`, however, so I will try to use that instead to maintain the status quo. We're facing similar ecosystem challenges in other InfluxData projects and replacing `1.4.x` with `1.3.x` has side-stepped this issue in the past.